clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1739,17 +1738,57 @@ class SyclKernelBodyCreator : public SyclKernelFieldHandler {
     InitExprs.push_back(MemberInit.get());
   }
 
+  int getDims() {
+    int Dims = 0;
+    for (int i = MemberExprBases.size() - 1; i >= 0; --i) {
+      if (!isa<ArraySubscriptExpr>(MemberExprBases[i]))
+        break;
+      ++Dims;
+    }
+    return Dims;
+  }
+
+  int64_t getArrayIndex(int i) {
```

Comment:
```suggestion
  int64_t getArrayIndex(int Idx) {
```

Reply:
Doesn't seem to be

Reply:
Sorry, I had to refresh my page.

Reply:
Hi Mariya. Can you check if you're checking final diff/commit? 

https://github.com/intel/llvm/pull/2341/files#diff-c88a590aca6d5f8e92dd1f8b6ec05a49R1786